website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,244 @@
+---
+name: 'ACL Replication for Multiple Datacenters '
+content_length: 15
+id: acl-replication
+layout: content_layout
+products_used:
+  - Consul
+description: You can configure tokens, policies, and roles to work across multiple datacenters.
+---
+
+You can configure tokens, policies, and roles to work across multiple
+datacenters. This provides central configuration of security policies in the
+primary datacenter and failover for the secondary datacenters. If the primary
+datacenter goes offline, the secondary datacenters will still have the security
+policies in place. 
+
+Note, each datacenter should have ACLs enabled and you need to complete the
+bootstrapping process outlined in the [Securing Consul with ACLs
+guide](/consul/security-networking/production-acls) on the primary datacenter.
+This guide includes the additional ACL replication configuration for the Consul
+agents. 
+
+In this guide, you will setup ACL replication. this is a multi-step process
+that includes:
+
+Setting the `primary_datacenter` parameter on all Consul agents in the primary
```

Comment:
bullet list?

Reply:
Yes! It must have lost formatting when I enforced wrapping.